assets/src/editor/events/tickets/editor-ticket/edit-form/ticket-entity-form.js

```diff
@@ -76,40 +59,82 @@ const TicketEntityForm = ( {
 		'wpUser',
 		'status',
 	];
-	const formRows = getFormRows(
-		ticketEntity,
-		calculator,
-		exclude,
-		currentValues,
-		FormInput,
-		recalculateBasePrice,
+	const inputConfig = useMemo(
+		() => ticketEntityInputConfig(
+			ticketEntity,
+			calculator,
+			recalculateBasePrice
+		),
+		[
+			ticketEntityInputConfig,
+			ticketEntity,
+			calculator,
+			recalculateBasePrice,
+		]
 	);
-	formRows.unshift(
-		<FormInfo
-			key="formInfo"
-			formInfo={
-				__(
-					'all fields marked with an asterisk are required',
-					'event_espresso'
-				)
-			}
-			dismissable={ false }
-		/>
+	const formRows = useMemo(
+		() => editEntityFormInputs(
+			ticketEntity,
+			inputConfig,
+			`ee-ticket-${ ticketEntity.id }`,
+			isEmpty( currentValues ) ? initialValues : currentValues,
+			exclude
+		),
+		[
+			editEntityFormInputs,
```

Comment:
`editEntityFormInputs` doesn't need to be a dependency here because it's defined outside of the component tree thus it doesn't change between renders.